src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -4,107 +4,75 @@ short-title: Supported platforms
-  are automatically tested at every commit
-  by continuous integration testing.
-1. Best-effort platforms, supported through community
+1. **Supported**, Google-tested platforms
+   are automatically tested at every commit
+   by continuous integration testing.
+1. **Best-effort** platforms, supported through community
    testing, are platforms we believe we support through
    coding practices and ad-hoc testing,
    but rely on the community for testing.
-1. Unsupported platforms, which are platforms that
+1. **Unsupported** platforms, which are platforms that
    might work, but that the development team
    doesn't directly test or support.
    
-## Supported platforms
-
-<!--
-*IMPORTANT NOTE*:
-When changing the minimum support version of macOS, Windows, and Linux,
+{% comment %}
+**IMPORTANT NOTE**:
+When changing the minimum support version of macOS, Windows, or Linux,
 make sure to make a corresponding change in the get started pages:
-  - /src/get-started/install/macos.md
-  - /src/get-started/install/windows.md
-  - /src/get-started/install/linux.md
--->
+  * /src/get-started/install/macos.md
+  * /src/get-started/install/windows.md
+  * /src/get-started/install/linux.md
+{% endcomment %}
 
-As of the current release,
-Flutter supports deployment of apps on the following platforms 
-as part of Google-tested and best-effort platform tier:
 
-<div class="table-wrapper" markdown="1">
-|Platform|Version                       |Channels |
-|--------|------------------------------|---------|
-|Android | API 16 (Android 4.1) & above | All     |
-|iOS     | iOS 11 & above               | All     |
-|Linux   | Debian, 64-bit               | All     |
-|macOS   | Mojave (10.14) & above       | All     |
-|Web     | Chrome 84  & above           | All     |
-|Web     | Firefox 72.0 & above         | All     |
-|Web     | Safari on El Capitan & above | All     |
-|Web     | Edge 1.2.0 & above           | All     |
-|Windows | Windows 7 & above, 64-bit    | All     |
-{:.table.table-striped}
-</div>
+## Deploying Flutter
 
-All channels include master, beta,
-and stable channels.
-
-### Google-tested platforms
+As of the current stable release,
+support for deploying Flutter apps is shown in the
+following table:
 
 <div class="table-wrapper" markdown="1">
-|Platform|Version               |
-|--------|----------------------|
-|Android |Android SDK 19–30*    |
-|iOS     |14-15                 |
-|Linux   |Debian 10             |
-|Linux   |Ubuntu 18.04 LTS      |
-|macOS   |Monterey (12) & above |
-|Web     |Chrome 84             |
-|Web     |Firefox 72.0          |
-|Web     |Safari / Catalina     |
-|Web     |Edge 1.2.0            |
-|Windows |Windows 10            |
+|Platform version        | Supported | Best effort | Unsupported |
+|------------------------|-----------|-------------|-------------|
+| Android SDK 19-30<sup>1</sup> | ✓  |             |             |
+| Android SDK 16-18      |           |      ✓      |             |
```

Comment:
Not that it matters hugely, but the checkmarks aren't centered horizontally in the output.